ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -127,7 +127,7 @@ function rhel_upgrade(){
     echo "Validating parsed Ansible inventory"
     ansible-inventory -i "${SHARED_DIR}/ansible-hosts" --list --yaml
     echo "Running RHEL worker upgrade"
-    ansible-playbook -i "${SHARED_DIR}/ansible-hosts" playbooks/upgrade.yml -vvv
+    ansible-playbook -i "${SHARED_DIR}/ansible-hosts" /usr/share/ansible/playbooks/upgrade.yml -vvv
```

Comment:
Is the path correct ?
```
[root@63a3a5083fc3 /]# ls -lhd /usr/share/ansible/openshift-ansible/playbooks/  /usr/share/ansible/playbooks/
ls: cannot access '/usr/share/ansible/playbooks/': No such file or directory
drwxr-xr-x. 1 root root 72 Apr  7 02:33 /usr/share/ansible/openshift-ansible/playbooks/
```

Reply:
ah, you're right. Thank you for the correction! Updated

Reply:
I just realized that maybe this path change need to happen together with the image switch, or else it might break existing RHEL upgrade as we haven't switched the image yet.

Reply:
That's right! Let me close it.